yb-voyager/cmd/importData.go

```diff
@@ -858,10 +858,11 @@ func cleanImportState(state *ImportDataState, tasks []*ImportFileTask) {
 		nonEmptyTableNames := lo.Map(nonEmptyNts, func(nt sqlname.NameTuple, _ int) string {
 			return nt.ForOutput()
 		})
-		utils.PrintAndLog("Following tables are not empty. "+
-			"TRUNCATE them before importing data with --start-clean.\n%s",
-			strings.Join(nonEmptyTableNames, ", "))
-		yes := utils.AskPrompt("Do you want to continue without truncating these tables?")
+		utils.PrintAndLog("Non-Empty tables: [%s]", strings.Join(nonEmptyTableNames, ", "))
+		utils.PrintAndLog("The above list of tables on target DB are not empty; " +
+			"partial data may have been imported in a previous run. Are you sure you want to start-clean? \n" +
+			"If you wish to truncate the tables, please manually truncate them on the target DB.")
```

Comment:
`"If you wish to start-clean, please manually truncate them on the target DB.")`

Reply:
This is a bit tricky :D 
>  If you wish to start-clean, 

In the previous line, we have "Are you sure you want to start-clean?"

Reply:
How about just "Please manually truncate them on the target DB, if required." ?